internal/handler/wallet/create_test.go

```diff
@@ -58,12 +59,16 @@ func Test_walletCreate_Handle(t *testing.T) {
 
 			assertWallet(t, nickname)
 
-			err = cleanupTestData([]string{nickname})
-			assert.NoError(t, err)
+			defer clean(t, []string{nickname})
```

Comment:
are you sure about this??
defer will be executed when function is returnning.
Previous version was executing cleanupTestData after each test case loop.

Reply:
for test case 'nickname already exists' I need to have a yaml file so that this test pass. But yeah it's not the best way to do it probably